content/docs/user-guide/gems/reference/multiplayer/multiplayer-debug/debug/desync-audit-trail.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Multiplayer Desync Audit Trail
+description: Analyze and Debug Multiplayer Desyncs
+linkTitle: Desync Audit Trail
+---
+
+When the Client and Server disagree on the value of a networked variable, a desync occurs. Desyncs can be extremely difficult to debug which is where the Multiplayer Desync Audit Trail can help.
+
+The Multiplayer Desync Audit Trail is a tool available with the Multiplayer Gem in Debug environments. It details all network desyncs that occur and categorizes network activity leading up to a desync to attempt to root cause the desync itself.
+
+![Audit Trail Overlay](/images/user-guide/gems/reference/multiplayer/audit_trail_default.png)
+
+The Audit Trail will capture a desync and the Input ID and Host Frame it occurred on. For each desync, it will list under it all captured activity leading up to the desync up to a CVAR defined limit.
+
+## Configuration
+Various CVARs can be modified to enable additional data capture. Enabling additional capture both adds a performance cost and density to the Audit Trail. If you require capturing a longer event history, consider raising *net_DebutAuditTrail_HistorySize*.
+
+| Property | Description | Type |
+|---|---|---|---|
+| cl_EnableDesyncDebugging | If true, enables output of Desyncs to standard logs AND Audit Trail capture. | `bool` |
+| cl_DesyncDebugging_AuditInputs | If true, adds Inputs to audit trail. | `bool` |
+| net_DebutAuditTrail_HistorySize | Maximum number of events the Audit Trail will aggregate. Raising this value is recommended if Input or Event Auditing is enabled to compensate for increased Event volume. | `int` |
+
+## Auditing Categories
+### Desync
+
+Desyncs are the primary event captured by the Audit Trail. Consequently the Audit Trail organizes all activity around Desyncs. On the client, desyncs generally include a delta map of variables the client and server disagreed on which led to the desync and their values for each endpoint. On the server, detailed data around client values is not available so the occurrence and timing of the desync are captured. 
```

Comment:
Feels like this could be shortened to "On the client, desyncs generally include a delta map of variables and the values the client and server disagreed on."